### PR TITLE
Created the .env file to store the token

### PR DESCRIPTION
**Pull request by aakash-cse**

### CODE REVIEW

**Comment by rexdivakar** [comment]
image
Why is this failing please check @aakash-cse

**Review by SanchitJain123 — CHANGES_REQUESTED** [review]
@aakash-cse it would be better if we could provide different token for Slack, Discord, and Telegram.

**Comment by aakash-cse** [comment]
> @aakash-cse it would be better if we could provide different token for Slack, Discord, and Telegram.

I don't know why it is failing since I haven't modified much of the working codr

**Comment by rexdivakar** [comment]
It is failing since the token is picked up as a whole string, Please add tokens for slack and telegram and run the test in parallel !! @aakash-cse 

image

**Comment by SanchitJain123** [comment]
> > @aakash-cse it would be better if we could provide different token for Slack, Discord, and Telegram.
> 
> I don't know why it is failing since I haven't modified much of the working codr

as @rexdivakar said it should be `r'` not `"r'`. Please try again after changing this

File: .env [hunk 1]
```diff
@@ -1 +1 @@
-TOKEN=r'https://discord.com/api/webhooks/771253998313144321/5FNBsAi8-exyc-3rxLOYudQdQqMdpcaCOqCh6z1IX4ledG1oGyaONeIY1HDUns6qBZhW'
+TOKEN=r"https://discord.com/api/webhooks/771253998313144321/5FNBsAi8-exyc-3rxLOYudQdQqMdpcaCOqCh6z1IX4ledG1oGyaONeIY1HDUns6qBZhW"
```
Comment (SanchitJain123) [inline .env:1]: try removing `r` itself

**Comment by SanchitJain123** [comment]
the problem now here is that the pytest is not using the environment variable.

**Comment by SanchitJain123** [comment]
sorry was busy for a while can we connect on discord @aakash-cse , message me on sanchit#5988

**Review by SanchitJain123 — APPROVED** [review]

**Comment by SanchitJain123** [comment]
thank you @aakash-cse

**Pull request merged**